docs/core/tools/dotnet-workload-install.md

```diff
@@ -0,0 +1,98 @@
+---
+title: dotnet workload install command
+description: The 'dotnet workload install' command installs optional workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload install
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
+
+## Name
+
+`dotnet workload install` - Installs optional workloads.
+
+## Synopsis
+
+```dotnetcli
+dotnet workload install <WORKLOAD_ID>
+    [--add-source <SOURCE>] [--configfile <FILE>] [--disable-parallel]
+    [--download-to-cache <CACHE>] [--from-cache <CACHE>]
+    [--ignore-failed-sources] [--include-previews] [--interactive]
+    [--no-cache] [--sdk-version <VERSION>] [--skip-manifest-update]
+    [--temp-dir <PATH>] [-v|--verbosity <LEVEL>]
+
+dotnet workload install -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload install` command installs an optional workload.
+
+## Arguments
+
+- **`WORKLOAD_ID`**
+
+  The workload ID of the workload to install.
+
+## Options
+
+<!-- markdownlint-disable MD012 -->
```

Comment:
Is this some false positive that can be reported to markdownlint?

Reply:
Possibly.  It's related to how includes are handled.  Include file has to end with a line feed, and then that makes it count two blank lines in the main file.

Reply:
Tagging @DavidAnson to take a look.

Reply:
Line 39 above would be a second blank line, so the markdownlint warning is correct. I don’t know how the file include process works in this project, but you may want to disable this rule or maybe only lint the merged output (though that would forfeit edit-time validation)?

Reply:
The problem rules that conflict so you can't satisfy both.  The include has to end in a blank line. The line that references the include has to be followed by a blank line because the following line is a bulleted list item. Remove the ending blank line in the include and you get a linter error.  Remove the line between the include reference and the following bulleted list item and you get a linter error.  There's no way out except to disable one of the rules.